src/XRP/network-clients/json-rpc-network-client.ts

```diff
@@ -0,0 +1,45 @@
+import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios'
+
+interface JsonRequest {
+  method: string
+  params: unknown
+}
+
+/**
+ * A network client for interacting with the rippled JSON RPC.
+ * @see https://xrpl.org/rippled-api.html
+ */
+export default class JsonRpcNetworkClient {
+  private readonly axiosInstance: AxiosInstance
+
+  /**
+   * Create a new JsonRpcNetworkClient.
+   *
+   * @param jsonUrl The URL of the rippled node to query.
+   * @see https://xrpl.org/get-started-with-the-rippled-api.html
+   */
+  public constructor(jsonUrl: string) {
+    this.axiosInstance = axios.create({
+      baseURL: jsonUrl,
+    })
+  }
+
+  /**
+   * Submit a JSON request to the rippled server.
+   *
+   * @param jsonRequest A JSON object specifying the method and params of the request.
+   * @see https://xrpl.org/request-formatting.html#json-rpc-format
+   * @returns The response from the rippled server.
+   */
+  public async submitRequest(
+    jsonRequest: JsonRequest,
+  ): Promise<AxiosResponse<unknown>> {
```

Comment:
can we abstract away `AxiosResponse`? this is leaking details about the underlying implementation, as the caller would have to know about `axios` to consume what this function returns

if you wanted to maintain an abstraction but still have flexibility for later, it may be helpful to do

```
type JsonRpcResponse<T> = AxiosResponse<T>
```

and then use `JsonRpcResponse` (or, ideally, a better name probably lol) instead of `AxiosResponse`

Reply:
yeah so I kinda realized this blurred boundary and in a subsequent PR, make it such that the `JsonRpcNetworkClient` will actually have a method for each specific RPC, much like the current gRPC-based network client does now (which is also super helpful for DI testing).  Therefore the `AxiosResponse` stays within the network client class and will just returns a JSON object that conforms to some expected type.